wiki/systems/soul-system.md

```
@@ -0,0 +1,85 @@
+created: 2026-04-09
+updated: 2026-04-09
+sources: [src/soul/SoulService.cs, src/soul/SoulModels.cs, src/soul/SoulExtractor.cs, src/soul/AgentProfile.cs, src/dream/autodreamservice.cs]
+---
+
+# Soul System
+
+Persistent identity, user understanding, project rules, and learning from mistakes/habits.
+
+## File Layout (under hermesHome)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)